app/Http/Requests/UserRegisterPostRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Contracts\Validation\Validator;
+use Notifications;
+use App\Http\Requests\Request;
+
+class UserRegisterPostRequest extends Request
+{
+
+
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        $rules = [
+            'name' => 'required|min:4|max:26',
+            'email' => 'required|email|unique:users,email|min:6|max:300',
+            'password' => 'required|confirmed|min:6|max:300',
+            'rules' => 'required',
```

Comment:
Yes, accepted.
If user checked rules checkbox - value is ON, if not - value is null.